homeassistant/components/advantage_air/climate.py

```diff
@@ -91,6 +91,14 @@ class AdvantageAirAC(AdvantageAirAcEntity, ClimateEntity):
     _attr_max_temp = 32
     _attr_min_temp = 16
 
+    _attr_hvac_modes = [
+        HVACMode.OFF,
+        HVACMode.COOL,
+        HVACMode.HEAT,
+        HVACMode.FAN_ONLY,
+        HVACMode.DRY,
+    ]
+
```

Comment:
You could add here `_attr_supported_features = ClimateEntityFeature.FAN_MODE` since this is supported on all cases.

Reply:
Changed as requested.